test/files.test.ts

```diff
@@ -141,6 +156,16 @@ npm run test -- --watch
 							expect(contents).toContain("In your terminal, run the");
 						}
 					});
+
+					test("tsconfig.json", () => {
+						const tsconfigData = readFileAsJSON(
+							`${chapterDirectory}/${projectSlug}/tsconfig.json`
+						);
+
+						expect(tsconfigData).toEqual({
+							include: ["src"],
+						});
+					});
```

Comment:
Todo: also verify in all meal types that they extend the root tsconfig. And for appetizers, that both the project directory _and_ the individual steps have TSConfigs.